iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -85,6 +85,7 @@ private InputFormatConfig() {
   public static final boolean CONFIG_SERIALIZATION_DISABLED_DEFAULT = true;
   public static final String OPERATION_TYPE_PREFIX = "iceberg.mr.operation.type.";
   public static final String OUTPUT_TABLES = "iceberg.mr.output.tables";
+  public static final String OUTPUT_TABLE_BRANCH = "iceberg.mr.output.table_branch";
```

Comment:
should we use dot "." separator to be consistent with other properties?

Reply:
Fixed.